public/index.html

```diff
@@ -6,6 +6,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <script src="https://kit.fontawesome.com/c53588373b.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="css/toggle-bootstrap.min.css" />
+    <link rel="stylesheet" href="css/toggle-bootstrap-dark.min.css" />
+    <link rel="stylesheet" href="css/toggle-bootstrap-print.min.css" />
```

Comment:
We need these stylesheets as files in our project, maybe under `public/css`. Otherwise they won't load.

Reply:
I could not find the file for "bootstrap-dark" anywhere on the internet, if you have it please send it to me.